packages/web-components/src/components/filter-panel/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,99 @@
+import { Preview, Props, Description, Story } from '@storybook/addon-docs/blocks';
+import '../filter-panel';
+import contributing from '../../../../../../docs/contributing-license.md';
+import { cdnJs, cdnCss } from '../../../globals/internal/storybook-cdn';
+
+
+
+
+
+# Filter Panel
+
+> 💡 Check our
+> [CodeSandbox](https://githubbox.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/web-components/examples/codesandbox/components/filter-panel)
```

Comment:
Mentioned in main comments that there currently isn't a codesandbox example. Can you add?